presto-yarn-test/src/test/groovy/com/teradata/presto/yarn/fulfillment/PrerequisitesClusterFulfiller.groovy

```diff
@@ -117,7 +118,26 @@ public class PrerequisitesClusterFulfiller
 
   private void restartResourceManager()
   {
-    runOnMaster([RESTART_RM_CMD])
+    runOnMaster(['/etc/init.d/hadoop-yarn-resourcemanager restart'])
+  }
+
+  private void setupCgroup()
+  {
+    runOnAll([
+            'ls /cgroup || yum install -y libcgroup',
```

Comment:
these things will need sudo access to the cluster you run tests on, right? Can you update the documentation saying this?